doc/dev/kdb-operations.md

```diff
@@ -22,33 +24,32 @@ The basic flow of this operation is:
 
 1. Create empty `KDB` instance
```

Comment:
I think this can be summarized more, otherwise it does not provide much benefit from reading the code.

Reply:
This was intended as specification, i.e. the target audience is contributors. The goal was a specification so that we know what must happen in code. For developers a separate summarized version needs to be created.